crates/holochain/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Adds the [`UseExisting`](https://github.com/holochain/holochain/blob/293d6e775b3f02285b831626c9911802207a8d85/crates/holochain_types/src/app/app_manifest/app_manifest_v1.rs#L155-L165) cell provisioning strategy, an alternative to `Create`, allowing an app to depend on a cell from a previously installed app. Read the rustdocs for more info on this new type of provisioning.
+- New protections are put in place for apps which are depended upon by other apps via `UseExisting`. Any "protected" inter-app dependency will prevent a dependency app from being uninstalled until the dependent app is also uninstalled, or if the `force` parameter is set to true in the `UninstallApp` call.
```

Comment:
I remember we had decided against such protections. Apps should just be uninstalled regardless of dependencies. I think the same should apply enabling/disabling.

Reply:
I remember the most recent conversation was that we would indeed have this protection, with a force flag. @ThetaSinner do you remember?

Reply:
As you say it. It shouldn't happen silently but the user should be able to deliberately break other apps by forcing the uninstall. That's how I remember it